packages/components/Molecules/Breadcrumbs/index.vue

```diff
@@ -34,7 +34,7 @@ export default {
 
   $rpl-breadcrumbs-background: rpl-color('white') !default;
   $rpl-breadcrumbs-border: 1px solid rpl-color('mid_neutral_1') !default;
-  $rpl-breadcrumbs-border-radius: rem(4px) !default;
+  $rpl-breadcrumbs-border-radius: var(--rpl-theme-border-radius, rem(4px)) !default;
```

Comment:
This might be better served by a SCSS variable - What if you wanted to change the default value everywhere? If you created a variable `$rpl-theme-border-radius: var(--rpl-theme-border-radius, rem(4px))` you would only have one place to change it.

Reply:
Ahhh true. I have thought about that when I asked you about the flexibility. I'll create a global variable. Thanks